src/templates/InstantSearch.js/.template.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   category: 'Web',
   libraryName: 'instantsearch.js',
   supportedVersion: '>= 3.0.0 < 5.0.0',
+  includesDynamicWidgets: "4.30",
```

Comment:
Instead of having a specific option for DynamicWidgets, we could have "flags" and provide the semver range to enable/disable them (instead of assuming `gte` in the CLI).

```diff
 module.exports = {
   category: 'Web',
   libraryName: 'instantsearch.js',
   supportedVersion: '>= 3.0.0 < 5.0.0',
-  includesDynamicWidgets: "4.30",
+  flags: {
+    dynamicWidgets: ">= 4.30.0"
+  },
   templateName: 'instantsearch.js',
   appName: 'instantsearch.js-app',
   keywords: ['algolia', 'InstantSearch', 'Vanilla', 'instantsearch.js'],
   tasks: {
     install,
     teardown,
   },
 };
```